javascript/javascript3/week3/homework/style.css

```diff
@@ -0,0 +1,65 @@
+*{
```

Comment:
It is not recommended to use the asterisk(*) selector. You can read a good explanation of why it is preferable to use the body tag instead in this post in Stack Overflow: https://stackoverflow.com/questions/53839401/is-the-css-asterisk-a-good-practice
Please move the styling from the * tag to the body tag.